### PR TITLE
print result object from kubectl taint correctly

### LINKED ISSUE #63767: Failing test: [sig-cli] Kubectl client tests failing in sig-release-master-blocking and master-upgrade
## Failing Job
* [gci-gce-serial](https://k8s-testgrid.appspot.com/sig-release-master-blocking#gci-gce-serial)
* [gci-gke-serial](https://k8s-testgrid.appspot.com/sig-release-master-blocking#gci-gke-serial)

## Failing Test
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gke-serial/6038

## Triage results
https://storage.googleapis.com/k8s-gubernator/triage/index.html?sig=cli&test=Kubectl%20client

This test is currently failing in sig-release master-blocking and upgrade suites since late 5/11. [Commit changes](https://github.com/kubernetes/kubernetes/compare/5133d19fb...5782d476c). This could potentially block Beta v1.11 which is due to be cut Tuesday,5/15. @liggitt can you please help triage?

/kind bug
/priority failing-test
/priority important-soon
/sig cli
@kubernetes/sig-cli-bugs 

cc @jberkus @tpepper

/assign @liggitt

### PR DESCRIPTION
**Pull request by liggitt**
fixes #63767 

```release-note
NONE
```

### CODE REVIEW

**Comment by liggitt** [comment]
@kubernetes/sig-cli-pr-reviews

**Review by soltysh — APPROVED** [review]
lgtm

**Review by juanvallejo — APPROVED** [review]

File: hack/make-rules/test-cmd-util.sh [hunk 1]
```diff
@@ -4617,6 +4617,17 @@ __EOF__
 }
 __EOF__
 
+  # taint/untaint
+  # Pre-condition: node has no taints
+  kube::test::get_object_assert "nodes 127.0.0.1" "{{.spec.taints}}" '<no value>'
+  # taint can add a taint
+  kubectl taint node 127.0.0.1 dedicated=foo:PreferNoSchedule
+  kube::test::get_object_assert "nodes 127.0.0.1" "{{.spec.taints}}" 'PreferNoSchedule'
```
Comment (juanvallejo) [inline hack/make-rules/test-cmd-util.sh]: .spec.taints returns a map

Comment (liggitt) [reply hack/make-rules/test-cmd-util.sh]: fixed test, retagging

**Pull request merged**